server/config/config.go

```diff
@@ -81,9 +81,13 @@ type BaseConfig struct {
 	// IndexEvents defines the set of events in the form {eventType}.{attributeKey},
 	// which informs Tendermint what to index. If empty, all events will be indexed.
 	IndexEvents []string `mapstructure:"index-events"`
+
 	// IavlCacheSize set the size of the iavl tree cache.
 	IAVLCacheSize uint64 `mapstructure:"iavl-cache-size"`
 
+	// IAVLDisableFastnNode enables or disables the fast sync node.
+	IAVLDisableFastnNode bool `mapstructure:"iavl-disable-fastnode"`
```

Comment:
nit: Is positive flags more readable? so we double have double negativities: `IAVLDisableFastnNode: false`.